{"identifier":{"url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonMalformed","interfaceLanguage":"occ"},"metadata":{"fragments":[{"text":"SMIChallengeReasonMalformed","kind":"identifier"}],"roleHeading":"Enumeration Case","symbolKind":"case","externalID":"c:@E@SMIChallengeReason@SMIChallengeReasonMalformed","role":"symbol","title":"SMIChallengeReasonMalformed","modules":[{"name":"SMIClientCore"}],"navigatorTitle":[{"text":"SMIChallengeReasonMalformed","kind":"identifier"}]},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"identifier","text":"SMIChallengeReasonMalformed"}],"languages":["occ"],"platforms":["iOS"]}]}],"hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason"]]},"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"symbol","abstract":[{"type":"text","text":"The passed in token was malformed."}],"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smichallengereason\/smichallengereasonmalformed"]}],"sections":[],"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason/SMIChallengeReasonMalformed":{"url":"\/documentation\/smiclientcore\/smichallengereason\/smichallengereasonmalformed","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonMalformed","fragments":[{"kind":"identifier","text":"SMIChallengeReasonMalformed"}],"kind":"symbol","abstract":[{"type":"text","text":"The passed in token was malformed."}],"type":"topic","title":"SMIChallengeReasonMalformed","role":"symbol"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason":{"kind":"symbol","role":"symbol","type":"topic","url":"\/documentation\/smiclientcore\/smichallengereason","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason","title":"SMIChallengeReason","abstract":[{"text":"The reason for a credential challenge. Possible values:","type":"text"}],"fragments":[{"kind":"identifier","text":"SMIChallengeReason"}]},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"role":"collection","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","url":"\/documentation\/smiclientcore","title":"SMIClientCore","kind":"symbol","type":"topic","abstract":[]}}}